thirdeye/thirdeye-frontend/app/styles/wrapper/styles.scss

```diff
@@ -31,12 +55,55 @@ body {
 }
 
 .spinner-wrapper {
-  margin: 24px;
+  margin: $te-default-element-spacing;
   height: 250px;
   position: relative;
 }
 
 .anomaly-title-bar {
   background: white;
   border-bottom: 1px #e9e9e9 solid;
-}
\ No newline at end of file
+}
+
+.te-search-results {
+
+  ul {
```

Comment:
nit: could we avoid tag selectors. this might be a good candidate for a Bem `block__element` class